court_scraper/base/captcha_solver.py

```diff
@@ -0,0 +1,65 @@
+from anticaptchaofficial.recaptchav2proxyless import *
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+import time
+
+# TODO: Support more captcha types.
+
+class CaptchaSolvers:
```

Comment:
Class name should match module name. I'd go with singular class name here rather than plural module name.

Reply:
Though on second thought, if the class provides interfaces for multiple captcha services, then the plural would be fine.